kedro/framework/cli/catalog.py

```diff
@@ -28,6 +28,7 @@ def _create_session(package_name: str, **kwargs: Any) -> KedroSession:
 
 
 def is_parameter(dataset_name: str) -> bool:
+    # TODO: when breaking change move it to kedro/io/core.py
```

Comment:
Is this todo still needed?

Reply:
Yes, as it makes sense to keep it there to use across other components